packages/docs/docs/tutorials/hello-world-part-1.md

```diff
@@ -3,59 +3,30 @@ sidebar_position: 4
 sidebar_label: Hello World App
 ---
 
-# Hello World Part 1
+# Hello World Part
```

Comment:
If we're dropping the `1`, shouldn't we just call this "Hello World"?

Reply:
Yeah, we should